gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/UpdateUngradedItemsPanel.java

```diff
@@ -218,6 +231,11 @@ public String getIdValue(final GbGroup g, final int index) {
 				new Object[]{"${score}", "${group}"})).setEscapeModelStrings(false));
 	}
 
+	private boolean isExtraCredit(Double grade, Assignment assignment, GbGradingType gradingType) {
+		return (GbGradingType.PERCENTAGE.equals(gradingType) && grade > 100) ||
```

Comment:
Yep, so this is the `grade` value entered by the user.  It comes in as a percentage and is applied as such.